tests/integrations/server/kv_service.rs

```diff
@@ -1744,3 +1746,207 @@ fn test_get_lock_wait_info_api() {
     must_kv_pessimistic_rollback(&client, ctx, b"a".to_vec(), 20);
     handle.join().unwrap();
 }
+
+#[test]
+fn test_api_version() {
+    const TIDB_KEY_CASE: &[u8] = b"t_a";
+    const TXN_KEY_CASE: &[u8] = &[key_prefix::TXN_KEY_PREFIX, 0, b'a'];
+    const RAW_KEY_CASE: &[u8] = &[key_prefix::RAW_KEY_PREFIX, 0, b'a'];
+
+    let test_data = vec![
```

Comment:
The APIv2 spec still confuse me. Can you explain the relation between APIv2 and RAW?

And I think it's better to add some comments about the spec here.

Reply:
Is this spec sufficient?
https://github.com/pingcap/kvproto/blob/d0692eebe1ed1827811b24a377a45b3bed58f15b/proto/kvrpcpb.proto#L715-L762